From 8e176c8b0d97e78e8b33fac3b5acded277bbcf4f Mon Sep 17 00:00:00 2001
From: Vlad Zahorodnii <vladzzag@gmail.com>
Date: Mon, 21 Oct 2019 21:20:11 +0300
Subject: Don't use MESA_EGL_NO_X11_HEADERS

Summary:
With recent Mesa updates (19.2.1), MESA_EGL_NO_X11_HEADERS doesn't have
any effect. So, let's include fixx11h.h instead.

Test Plan: I am able to build kwin once again.

Reviewers: #kwin, davidedmundson

Reviewed By: #kwin, davidedmundson

Subscribers: kwin

Tags: #kwin

Differential Revision: https://phabricator.kde.org/D24840
---
 plugins/qpa/eglhelpers.h       | 2 +-
 plugins/qpa/offscreensurface.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/qpa/eglhelpers.h b/plugins/qpa/eglhelpers.h
index 1eb7624..30eb56d 100644
--- a/plugins/qpa/eglhelpers.h
+++ b/plugins/qpa/eglhelpers.h
@@ -20,9 +20,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-#define MESA_EGL_NO_X11_HEADERS
 #include <epoxy/egl.h>
 #include "fixqopengl.h"
+#include <fixx11h.h>
 
 #include <QSurfaceFormat>
 
diff --git a/plugins/qpa/offscreensurface.h b/plugins/qpa/offscreensurface.h
index 1254981..5b8ce29 100644
--- a/plugins/qpa/offscreensurface.h
+++ b/plugins/qpa/offscreensurface.h
@@ -20,9 +20,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-#define MESA_EGL_NO_X11_HEADERS
 #include <epoxy/egl.h>
 #include "fixqopengl.h"
+#include <fixx11h.h>
 
 #include <qpa/qplatformoffscreensurface.h>
 
-- 
cgit v1.1
